src/index.js

```diff
@@ -153,6 +153,20 @@ if (okToStart) {
   audit(inputData);
 }
 
+// We assume that each input line contains at least 3 comma-separated values.
+// Only the last value - URL - may contain commas.
+// We join all items after two first if the array of data contains more then 3 items.
+function getUrl(page) {
+  let url;
+  const pageParts = page.split(',');
+  if (pageParts.length > 3) {
+	url = pageParts.slice(2, pageParts.length).join();
+  } else {
+   url = pageParts[2];
+  }
```

Comment:
Nice!

Even simpler might just be to get `pageParts.slice(2, pageParts.length).join();` for every page.

Don't think that will affect performance.

In other words, in `audit()`:

```
const pageParts = page.split(',');
// First two pageParts are website name and page name. 
// URL may contain commas.
const url = pageParts.slice(2, pageParts.length).join();
```
(Or put this in a `getURL(page)` function.)

Reply:
Great, thank you! Made a change to the `getURL(page)` function as you reccommended.